packages/workflows/src/handlers/product/attach-sales-channel-to-products.ts

```diff
@@ -10,7 +9,7 @@ export async function attachSalesChannelToProducts({
   data,
 }: WorkflowArguments<{
   productsHandleSalesChannelsMap: Map<ProductHandle, SalesChannelId[]>
-  products: ProductTypes.ProductDTO[]
+  products: { handle: string; id: string }[]
```

Comment:
yaas, i like this. I ended up wanting to do the same thing. 

suggestion: Maybe put this as a type at the beginning of the file?

like: `WorkflowArguments<HandlerOutputData>`

Reply:
I ve extracted a portion of it, but yeah I can do that 👍